internal/operators/mce/config.go

```diff
@@ -15,3 +20,28 @@ const (
 type Config struct {
 	MceMinOpenshiftVersion string `envconfig:"MCE_MIN_OPENSHIFT_VERSION" default:"4.10.0"`
 }
+
+func GetMinDiskSizeGB(cluster *models.Cluster) int64 {
+	if common.IsSingleNodeCluster(&common.Cluster{Cluster: *cluster}) {
+		return sum(storageSizeGi)
+	}
+	return max(storageSizeGi)
+}
+
+func max(storageSize map[string]int64) int64 {
+	var maxValue int64
+	for _, v := range storageSize {
+		if v > maxValue {
+			maxValue = v
+		}
+	}
+	return maxValue
+}
+
+func sum(storageSize map[string]int64) int64 {
+	var total int64
+	for _, v := range storageSize {
+		total += v
+	}
+	return total
+}
```

Comment:
Instead of max/sum we can use go-funk or [lo](https://github.com/samber/lo)

Reply:
amazing, we probably should replace funk with lo, I don't really like having them both around.
Or maybe just wait to upgrade to go 1.22 where most uses will be covered by stdlib

Reply:
Which part of go 1.22 will cover most uses?